releasenotes/notes/add-sql-obfuscator-options-and-update-cgo-bindings-078a7fe8bf91076e.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    APM: Adds support to provide options to the SQL obfuscator to change the behavior.
```

Comment:
Not really. Users can’t really do anything new with this change. The only change happens on the python side and that’s what should be mentioned